### PR TITLE
Drop top-level checkoutLine query with related resolver

### PR DESCRIPTION
**Pull request by dexon44**
I want to merge this change because it drop deprecated query.

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [X] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [X] GraphQL schema and type definitions are up to date
* [X] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3832 Drop `checkoutLine` query from the schema](https://app.clickup.com/t/2549495/SALEOR-3832)

**Review by maarcingebala — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -226,6 +226,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Unify checkout identifier in checkout mutations and queries - #7511 by @IKarbowiak
 - Use root level channel argument for filtering and sorting - #7374 by @IKarbowiak
   - drop `channel` field from filters and sorters
+- Drop top-level `checkoutLine` query from the schema with related resolver - #3832 by @dexon44
```
Comment (tomaszszymanski129) [inline CHANGELOG.md:229]: It's good to mention what to use instead, search `instead` in this file for example.

**Review by tomaszszymanski129 — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (maarcingebala) [reply CHANGELOG.md:229]: In this case, we don't have a 1:1 replacement, but we can say that the `checkout` query should be used instead. We want people to get the checkout, and from there fetch the lines.

Comment (maarcingebala) [reply CHANGELOG.md:229]: CC @dexon44 ⬆️

File: CHANGELOG.md [hunk 2]
```diff
@@ -226,6 +226,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Unify checkout identifier in checkout mutations and queries - #7511 by @IKarbowiak
 - Use root level channel argument for filtering and sorting - #7374 by @IKarbowiak
   - drop `channel` field from filters and sorters
+- Drop top-level `checkoutLine` query from the schema with related resolver, use `checkout` query instead - #3832 by @dexon44
```
Comment (tomaszszymanski129) [inline CHANGELOG.md:229]: ⚠️ The number after `#` sign should be PR number - `#7623` in your case, not the ticket number.

**Review by tomaszszymanski129 — CHANGES_REQUESTED** [review]

**Review by tomaszszymanski129 — APPROVED** [review]

**Pull request merged**